lib/dry/web/roda/templates/Rakefile.tt

```diff
@@ -15,11 +15,30 @@ def db
   <%= config[:camel_cased_app_name] %>::Container["persistence.db"]
 end
 
+def settings
+  <%= config[:camel_cased_app_name] %>::Container["settings"]
+end
+
+def execute_database_create_command(settings)
+  database_url = settings.database_url
+  _, _, _, db_name = database_url.split('/')
```

Comment:
I think it might be safer if we could use `URI.parse` for this instead of trying to split on a character?

That'd also let us extract the username and password cleanly, too.